src/openzeppelin/tests/introspection/test_src5.cairo

```diff
@@ -1,7 +1,7 @@
-use openzeppelin::introspection::src5::SRC5;
-use openzeppelin::introspection::src5::SRC5::SRC5Impl;
-use openzeppelin::introspection::src5::SRC5::InternalImpl;
 use openzeppelin::introspection::interface::ISRC5_ID;
+use openzeppelin::introspection::src5::SRC5::InternalImpl;
+use openzeppelin::introspection::src5::SRC5::SRC5Impl;
+use openzeppelin::introspection::src5::SRC5;
```

Comment:
mmmm, good point. For some reason that's the default behavior when comparing the strings. I think there's no need to force the order the other way tbh

Reply:
I mean...yeah, I suppose we don't _need_ to force the order. I may try and fix this in the future because it's going to drive me crazy haha but at least it's consistent

Reply:
Got you :). Although, I think is gonna help marto, because is kind of sorting by line length as he was doing before